### PR TITLE
REALMC-7429: Finalize CLI help text

### PR DESCRIPTION
**Pull request by makesitgo**
looking for all kinds of thoughts/suggestions here.  a bulk of these changes are centered around:
* sorting the commands/flags so they appeared logically and consistently
* trying to keep language consistent across the various command texts
* removing most shorthands for flags, some of them became pretty unintuitive (e.g. `s` for `include-hosting`??) and i think the better approach here long-term would be figuring out proper auto-complete
* various command text cleanups with regards to spacing and such

### CODE REVIEW

File: internal/commands/app/delete_inputs.go [hunk 1]
```diff
@@ -11,8 +11,9 @@ import (
 )
 
 var (
-	flagApps      = "apps"
-	flagAppsUsage = "the Realm app names or ids to manage"
+	flagApp      = "app"
```
Comment (kpatel71716) [inline internal/commands/app/delete_inputs.go:14]: 👍  I like this change + having a short

File: internal/cli/inputs.go [hunk 2]
```diff
@@ -32,8 +34,10 @@ func (i ProjectInputs) Filter() realm.AppFilter { return realm.AppFilter{i.Proje
 
 // Flags registers the project app input flags to the provided flag set
 func (i *ProjectInputs) Flags(fs *pflag.FlagSet) {
+	fs.StringVarP(&i.App, flagApp, flagAppShort, "", flagAppUsage)
+
 	fs.StringVar(&i.Project, flagProject, "", flagProjectUsage)
-	fs.StringVar(&i.App, flagApp, "", flagAppUsage)
+	flags.MarkHidden(fs, flagProject)
```
Comment (kpatel71716) [inline internal/cli/inputs.go:40]: [q] Is there a reason the project flag is to be hidden? I imagine there would be a use case where someone would want to see apps filtered by project

**Review by kpatel71716 — COMMENTED** [review]
Looks really good! Just have one question from this pass

File: internal/cli/inputs.go (hunk 2 above)
Comment (makesitgo) [reply internal/cli/inputs.go:40]: historically that project flag was really only useful for mongo employees (who have access to _a lot_ of apps).  in a similar vein to the base urls for realm/atlas, I made them hidden.

**Review by nathantfrank — APPROVED** [review]
LGTM, pretty slick stuff :metal:

**Pull request merged**